fdbserver/Resolver.actor.cpp

```diff
@@ -249,6 +248,9 @@ ACTOR Future<Void> resolveBatch(Reference<Resolver> self, ResolveTransactionBatc
 		int64_t stateMutations = 0;
 		int64_t stateBytes = 0;
 		LogPushData toCommit(self->logSystem); // For accumulating private mutations
+		std::unordered_map<ptxn::TLogGroupID, std::shared_ptr<ptxn::ProxySubsequencedMessageSerializer>>
+		    groupMessageBuilders;
+		toCommit.pGroupMessageBuilders = &groupMessageBuilders;
```

Comment:
why not directly set `pGroupMessageBuilders` in LogPushData to be `std::unordered_map`?

Reply:
The commit proxy has the `std::unordered_map` defined in `struct CommitBatchContext` and passed in the same way. So we can't just declare in `LogPushData `.